src/main/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfiguration.java

```diff
@@ -64,7 +64,8 @@
 /**
  * Store the AWS configuration to save it on a separate file
  */
-@Extension @Symbol("credentials")
+@Symbol("awsCredentials")
```

Comment:
While `credentials` actually looks a bit nicer in the content of JEP-201, it is illegal for `@Symbol` generally as the `ExtensionList` here is `GlobalConfiguration`, not `AbstractAwsGlobalConfiguration`, so that would (probably) not be unique.

Reply:
From JCasC point of view the symbol scope is the GlobalConfiguration category, so "credentials" is fine. But all yours.

Reply:
The issue is that it is ambiguous what a symbol scope will be. The Javadoc only promises that

> symbol names are meant to be only unique within a specific extension point

which is fairly vague. Currently `configuration-as-code` does restrict the scope by category (I suppose in `GlobalConfigurationCategoryConfigurator.describe`), meaning it actually uses a different interpretation of the specification (since a `GlobalConfigurationCategory` is a classification but not an “extension point”). Other tools hewing more closely to the spec might call something like

```java
SymbolLookup.find(GlobalConfiguration.class, symbol)
```

since that type is where the `ExtensionPoint` is declared.

Reply:
indeed